source/indexes/create-vector-search-index.txt

```diff
@@ -0,0 +1,142 @@
+.. _compass-create-vector-search:
+
+===================================
+Create an Atlas Vector Search Index
+===================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 1.42.1
+
+You can create Atlas Vector Search indexes using |compass|. These indexes 
+enable you to index vector data *and* other data types, facilitating 
+semantic searches on the indexed fields. Atlas Vector Search indexes support 
+indexing vector data by identifying the most similar vectors. This is achieved 
+by calculating the distance between the query vector and the vectors stored 
+in the index.
+
+About this Task
+---------------
+
+To create an Atlas Search index, your deployment must be either:
```

Comment:
[suggestion] to [avoid introducing a list with a fragment ](https://www.mongodb.com/docs/meta/style-guide/style/lists/#introductory-text)

```suggestion
To create an Atlas Search index, your deployment must align with one of the following cases:
```

Reply:
updated.